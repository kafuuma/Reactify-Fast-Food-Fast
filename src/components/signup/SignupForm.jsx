import React from 'react';
import {
  MDBContainer,
  MDBRow,
  MDBCol,
  MDBInput,
  MDBBtn,
  MDBCard,
  MDBCardBody
} from 'mdbreact';

const SignupForm = ({ handleSubmit, handleChange, message }) => {
  return (
    <MDBContainer>
      <MDBRow className="d-flex flex-center pt-5">
        <MDBCol md="6">
          <MDBCard>
            <MDBCardBody>
              <form onSubmit={handleSubmit}>
                <p className="h4 text-center py-4">Sign up</p>
                <div className="grey-text">
                  <MDBInput
                    name="full_name"
                    onChange={handleChange}
                    label="Your name"
                    icon="user"
                    group
                    type="text"
                    validate
                    error="wrong"
                    success="right"
                  />
                  <MDBInput
                    name="email"
                    onChange={handleChange}
                    label="Your email"
                    icon="envelope"
                    group
                    type="email"
                    validate
                    error="wrong"
                    success="right"
                  />
                  <MDBInput
                    name="contact"
                    onChange={handleChange}
                    label="contact"
                    icon="envelope"
                    group
                    type="text"
                    validate
                    error="wrong"
                    success="right"
                  />
                  <MDBInput
                    name="password"
                    onChange={handleChange}
                    label="Your password"
                    icon="lock"
                    group
                    type="password"
                    validate
                  />
                  <MDBInput
                    name="confirm_password"
                    onChange={handleChange}
                    label="Confirm your email"
                    icon="exclamation-triangle"
                    group
                    type="password"
                    validate
                    error="wrong"
                    success="right"
                  />
                </div>
                <div className="text-center py-4 mt-3">
                  <MDBBtn id="bg" type="submit">
                    Register
                  </MDBBtn>
                </div>
              </form>
            </MDBCardBody>
          </MDBCard>
        </MDBCol>
      </MDBRow>
    </MDBContainer>
  );
};

export default SignupForm;
