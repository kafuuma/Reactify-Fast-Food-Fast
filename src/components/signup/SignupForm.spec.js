import React from 'react';
import { shallow } from 'enzyme';
import SignupForm from './SignupForm';

describe('<SignupForm />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SignupForm />);
  });

  it('matches snapshort', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
