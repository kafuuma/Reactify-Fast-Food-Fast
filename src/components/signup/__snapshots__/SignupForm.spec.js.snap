// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignupForm /> matches snapshort 1`] = `ShallowWrapper {}`;
